facebookresearch/beanmachine
# Docs: Adds sphinx to repo. Updates icon colors and theme.

### Motivation
Adds Sphinx API documentation to the repo. Also updates the site icons and colors. 

### Changes proposed
Sphinx content generated to /sphinx. New cosmetic content in /website. No changes to source code. 

### Test Plan
To test the sphinx content:

```
cd sphinx
pip install -r requirements.txt
make html. 
```

to see the changes to website:
```
cd website
yarn
yarn start
```

### Types of changes

### Checklist
- [x] I have read the **[CONTRIBUTING](https://github.com/facebookincubator/BeanMachine/blob/master/CONTRIBUTING.md)** document.
